lib/utils/docker.coffee

```diff
@@ -150,7 +150,8 @@ exports.tarDirectory = tarDirectory = (dir) ->
 		relPath = path.relative(path.resolve(dir), file)
 		Promise.join relPath, fs.stat(file), fs.readFile(file),
 			(filename, stats, data) ->
-				pack.entryAsync({ name: filename, size: stats.size, mode: stats.mode }, data)
+				posixFilename = filename.replace(new RegExp('\\' + path.sep, 'g'), '/')
```

Comment:
Could you pull this conversion out into a separate function, so these become something like `toPosixPath(filename)`? It'd be nice to make it clearer why this is required.